haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -682,6 +682,13 @@ instance CustomErrorHasDoc "nOT_OWNER" where
   customErrDocMdCause =
     "The sender of transaction is not owner"
 
+type instance ErrorArg "oPERATION_PROHIBITED" = NoErrorArg
```

Comment:
:thinking: It seems that this is also missing from the `specification.md`, can you add it there please?